.github/workflows/pre-commit.yaml

```diff
@@ -16,4 +16,5 @@ jobs:
         python -m ensurepip
         python -m pip install --upgrade pip
         python -m pip install .[dev]
+        npm install --prefix dtbase/webapp/ --include=dev
```

Comment:
Also, if this is needed, we should probably add a similar note to the developer documentation, at the same point when we advice people how to use pre-commit. (I'm assuming we say something about pre-commit in the dev docs.)

Reply:
I agree, though do you want me to do this right now or leave it open for later when you have finished with the other PR? I am not fully aware of how you are restructuring all the documentation

Reply:
If you want to add a comment in the dev docs feel free to go ahead, I can handle merging it with the docs rewrite. If you are busy we can merge this as is and I'll add the comment later.

Reply:
Sure, I will do that